app/components/public/ticket-list.js

```diff
@@ -29,14 +29,20 @@ export default Component.extend(FormMixin, {
   }),
   hasTicketsInOrder: computed('tickets.@each.orderQuantity', function() {
     return sumBy(this.tickets.toArray(),
-      ticket => ticket.getWithDefault('orderQuantity', 0)
+      ticket => (ticket.orderQuantity || 0)
     ) > 0;
   }),
 
   total: computed('tickets.@each.orderQuantity', 'tickets.@each.discount', function() {
-    return sumBy(this.tickets.toArray(),
-      ticket => (ticket.getWithDefault('price', 0) - ticket.getWithDefault('discount', 0)) * ticket.getWithDefault('orderQuantity', 0)
-    );
+    if (this.taxInfo.get('name') !== undefined) {
+      return sumBy(this.tickets.toArray(),
+        ticket => (ticket.ticketPriceWithTax || 0) * (ticket.orderQuantity || 0)
+      );
+    } else {
+      return sumBy(this.tickets.toArray(),
+        ticket => ((ticket.price || 0) - (ticket.discount || 0)) * (ticket.orderQuantity || 0)
+      );
```

Comment:
Else block here isn't needed, if `if` passes then it will `return` that means the code will just execute forward in case if doesn't pass, simply writing `return` instead of else should do.
```javascript
if (this.taxInfo.get('name') !== undefined) {
      return sumBy(this.tickets.toArray(),
        ticket => (ticket.ticketPriceWithTax || 0) * (ticket.orderQuantity || 0)
      );
    } 
    return sumBy(this.tickets.toArray(),
      ticket => ((ticket.price || 0) - (ticket.discount || 0)) * (ticket.orderQuantity || 0)
      );
```